dev-app/routes/components/pill/properties/index.html

```diff
@@ -57,6 +57,33 @@
 
         <c-divider></c-divider>
 
+        <c-divider></c-divider>
+
+        <div>
+            <l-stack spacing="var(--s-3)">
+                <c-h3>Font-Family, Font &amp; Link color</c-h3>
+                <c-code-sample>
+                    <c-cluster align="start">
+                        <div>
+                            <c-h3>Text in a paragraph</c-h3>
+                            <c-pill text-color="var(--c_black)" color="var(--c_secondaryMain)" font-family="HGD_75">
```

Comment:
for the value of `font-family` can we put the css var value in there instead? 
`font-family:  var(--ff_primary-bold)`